### PR TITLE
Create amazon-redshift.mdx

### PR DESCRIPTION
**Pull request by souravpy**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by souravpy** [comment]
Kindly merge this edit, if it needs any revisions I will take it from there, I have tried my best to implement the documentation and following all instructions.

File: docs/data-integrations/amazon-redshift.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Amazon Redshift
+sidebarTitle: amazon-redshift.mdx
```
Comment (martyna-mindsdb) [inline docs/data-integrations/amazon-redshift.mdx:3]: Please replace `amazon-redshift.mdx` with `Amazon Redshift`.

It should look like this:
```
---
title: Amazon Redshift
sidebarTitle: Amazon Redshift
---
```

File: docs/data-integrations/amazon-redshift.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* `user`: the user to authenticate the user with the Redshift cluster
+* `password`: the password to authenticate the user with the Redshift cluster
+
+## Usage
+Before attempting to connect to a Redshift cluster using MindsDB, ensure that it accepts incoming connections. The following can be used as a guideline to accomplish this,
+<br>
+https://aws.amazon.com/premiumsupport/knowledge-center/cannot-connect-redshift-cluster/
+
+In order to make use of this handler and connect to a Redshift cluster in MindsDB, the following syntax can be used,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/amazon-redshift.mdx:30]: Replace `~~~~sql` with ```sql

**Comment by souravpy** [comment]
Thanks for all the remarks, I am on it.

**Comment by martyna-mindsdb** [comment]
Closing https://github.com/mindsdb/mindsdb/issues/4460

**Pull request merged**